stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/avatar/AvatarStyle.kt

```diff
@@ -19,9 +20,8 @@ public data class AvatarStyle(
 ) {
 
```

Comment:
A big chunk of work is left here if we want to customize `AvatarView` for each particular view.